app/assets/stylesheets/components/_supplier-table.scss

```diff
@@ -26,8 +16,16 @@
 }
 
 .supplier-table--show-more {
+  padding-bottom: 110px;
+
+  @include cads-media-breakpoint-up(md) {
+    padding-bottom: 0;
+  }
+
   .js-show-more-suppliers {
     display: block;
+    position: absolute;
+    margin-top: 45px;
```

Comment:
I think we should rearrange the `haml` so that we don't need to break the default document flow to achieve this positioning.  Doing it this way has introduced:
- an invisible area to the right of the text and button on mobile that actually scrolls the table, this is a usability issue and unexpected behaviour
- the need to have additional margins on this element and padding on the table, which will be a bit confusing if we want to come and change this later
- the need to have styling rules (beyond showing / hiding) on a `.js-*` class.  These selectors are intended to be javascript hooks and applying these sort of styling rules breaks convention of keeping behavioural selectors for behaviour only

It would be better to bring the `p` and two `button` tags outside of `.cads-table-container` then you can keep the normal flow and get rid of the weird paddings/margins

Reply:
Sure, I think Mariana's solution addresses this so I'll implement it.

Reply:
now removed